### PR TITLE
py-tmuxp: new submission

### PR DESCRIPTION
**Pull request by egorenar**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.1 18B75
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-tmuxp/Portfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+maintainers         {@egorenar posteo.net:egorenar-dev} openmaintainer
+
+description         tmux session manager.
+long_description    ${description}
+
+homepage            https://github.com/tmux-python/tmuxp/
+master_sites        pypi:t/${python.rootname}/
+distname            ${python.rootname}-${version}
+python.versions     27 34 35 36 37
```
Comment (mf2k) [inline python/py-tmuxp/Portfile]: Please remove the old py34 and py35 versions.

Comment (egorenar) [reply python/py-tmuxp/Portfile]: fixed.

**Comment by egorenar** [comment]
i added setuptools as a runtime dependency because it is required at runtime.

**Comment by egorenar** [comment]
bump, any issues with this submission ?

**Pull request merged**